src/components/Footer/index.tsx

```diff
@@ -51,6 +85,75 @@ export const Footer: React.FC<FooterType> = props => {
     }
   }, [])
 
+  const router = useRouter()
+
+  const pathname = usePathname()
+
+  const onSubmit = React.useCallback(() => {
+    setButtonClicked(false)
+    const submitForm = async () => {
+      setError(undefined)
+
+      try {
+        const formID = process.env.NEXT_PUBLIC_NEWSLETTER_FORM_ID
+        const hubspotCookie = getCookie('hubspotutk')
+        const pageUri = `${process.env.NEXT_PUBLIC_SITE_URL}${pathname}`
```

Comment:
Is there a slash between these from either the env var or usePathname?

Reply:
`pathname` returns with a slash, yes.